eZ/Publish/Core/SignalSlot/ContentService.php

```diff
@@ -648,14 +660,17 @@ public function loadTranslationInfos(ContentInfo $contentInfo, array $filter = a
      * @param \eZ\Publish\API\Repository\Values\Content\ContentInfo $contentInfo
      * @param string $languageCode
      *
-     * @since 6.11
+     * @since 6.13
      */
-    public function removeTranslation(ContentInfo $contentInfo, $languageCode)
+    public function deleteTranslation(ContentInfo $contentInfo, $languageCode)
     {
-        $this->service->removeTranslation($contentInfo, $languageCode);
+        $this->service->deleteTranslation($contentInfo, $languageCode);
         $this->signalDispatcher->emit(
```

Comment:
maybe mark this as deprecated to, and ignore what I said about listening to both in ezplatform-http-cache and solr, given both is emitted that will cause it to trigger it twice.

Maybe only emit this signal from within `removeTranslation ` after call to `deleteTranslation` ?

Reply:
Hmm, it's a little bit git diff mess, `removeTranslation` is still available above, inheriting doc from deprecated interface method.

> Maybe only emit this signal from within removeTranslation after call to deleteTranslation ?

BC break :)
It's quite possible to use new API but still rely on old Signal. The perfect example is Solr Bundle. Initially I emitted `RemoveTranslationSignal` from deprecated method only, which failed on Solr as updated test for it is in kernel, but Solr Search Slot expects `RemoveTranslationSignal`.

Reply:
Ok, so we will need to wait for 1.13 to be out before we can change packages then, or?

Reply:
Hmm, can't we change it in `dev-master` of packages once this one is merged?

Reply:
yes, but we can't release it